compiler/ilgen/MethodBuilder.cpp

```diff
@@ -358,11 +358,11 @@ MethodBuilder::lookupSymbol(const char *name)
 
    TR::DataType type = ((TR::IlType *)(_symbolTypes->getData(typesID)))->getPrimitiveType();
 
-   TR_HashId slotID;
-   if (_parameterSlot->locate(name, slotID))
+   std::map<const char *, int32_t>::iterator it = _parameterSlot.find(name);
+   if (it != _parameterSlot.end())
       {
       symRef = symRefTab()->findOrCreateAutoSymbol(_methodSymbol,
-                                                   (int32_t)(uintptr_t)_parameterSlot->getData(slotID),
+                                                   it->second,
```

Comment:
Although I set a bad example here by directly using "getData()"... I confess, I am not a fan of using `it->second` directly. I would prefer you put it in a meaningful variable name to make it more clear (to me, I suppose) and rely on the compiler to optimize the renaming away.